src/System/AdminConfig.php

```diff
@@ -57,7 +57,18 @@ public static function get(): array
     {
         $repository = self::getRepository();
 
-        return SystemConfig::getConfigDataByKey($repository, self::CONFIG_ID);
+        $data = SystemConfig::getConfigDataByKey($repository, self::CONFIG_ID);
+        $loadType = $repository->getReadTargets()[0] ?? null;
+
+        // If the read target is settings-store and no data is found there,
+        // load the data from the container config
+        if(!$data && $loadType === $repository::LOCATION_SETTINGS_STORE) {
+            $containerConfig = \Pimcore::getContainer()->getParameter('pimcore_admin.config');
```

Comment:
can we get this data from `$repository->getDataFromContainerConfig(self::CONFIG_ID)`?

Reply:
No, because this method 'getDataFromContainerConfig()' has private visibility

Reply:
Got it. thanks!